### ISSUE TITLE
[SUGGESTION] Custom currency at Community Market

### ISSUE THREAD

**Issue by ardendanmer** [jshackles/Enhanced_Steam#1487]
It would be really neat to see pricing for items in USD. Market prices in my recently added currency are confusing and it takes way to much time to figure out what is a good deal.

**Comment by ntien93** [comment]
Yeah, local currency have too much decimal numbers makes Steam market so much confusion. It would be really nice to search/list/buy items in USD or just simply show them in USD

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
